_posts/2016/2016-03-16-team-orange-mallorca.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "A team of ePagees conquers Tierra Buena in Mallorca"
+date: "2016-03-16 06:16:03"
+icon: suitcase
+categories: working-at-epages
+authors: ["Alejandra A., Benjamin N. and Heiko A."]
+---
+
+Last week Team Orange, the ePages release and test automation team, was welcomed by the precious views, hearty people and refreshing atmosphere of Mallorca. The purpose of this particular trip was to celebrate an off-site team event liberated from the daily tasks and routines of the business life. The team set off to new shores to develop a master plan on how it could face the winds of change of the advancement of each team member. The ultimate goal was to find the best way for contributing to the promising future of the whole research and development department.
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. So please enjoy the ambigous report!
+
+{% image blog/blog-mallorca-1.jpg %}{% endimage %}
+
+With much curiosity we first took some time and visited the Drach Caves and the beautiful beach at Porto Cristo and recharged ourselves with inspiration.
+We also visited Sa Calobra and were delighted by the majestic mountains and the bluish tonalities of the Mediterranean Sea.
+
+{% image blog/blog-mallorca-2.jpg %}{% endimage %}
+
+Sharing daily life activities such as having breakfast, shopping, cooking, travelling by car and plane, playing board games and even doing sports together was definitely a delightful way to become closer. Our off-site time was a **BIG** chance to gain trust into the capabilities of our team. We totally adopted this magical present. Unsurprisingly, we soon recognized that living the team spirit can be a lot of fun!
+
+{% image blog/blog-mallorca-3.jpg %}{% endimage %}
+
+There is always time for everything and so we had enough time to work on our technological and business challenges, too. We accomplished two knowledge sharing sessions, one by our pipeline expert in order to understand how it works, another one on language handling and language packs.
```

Comment:
There is always time for everything and so we had enough time to work on our technological and business challenges, too.  > During our time on Mallorca, we of course had lots of time to work on our technological and business challenges, too.